site/package.json

```diff
@@ -27,7 +27,7 @@
     "prism-react-renderer": "^1.3.5",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
-    "trim": "0.0.3",
+    "trim": "0.0.1",
```

Comment:
Why was this downgraded? I'm pretty sure the version bump was done earlier for a security fix.